client/manager_sql.go

```diff
@@ -21,12 +34,75 @@ type SQLManager struct {
 }
 
 type sqlData struct {
-	ID      string `db:"id"`
-	Version int    `db:"version"`
-	Client  []byte `db:"client"`
+	ID                string `db:"id"`
+	Name              string `db:"client_name"`
+	Secret            string `db:"client_secret"`
+	RedirectURIs      string `db:"redirect_uris"`
+	GrantTypes        string `db:"grant_types"`
+	ResponseTypes     string `db:"response_types"`
+	Scope             string `db:"scope"`
+	Owner             string `db:"owner"`
+	PolicyURI         string `db:"policy_uri"`
```

Comment:
I'd divide this table into two - one with OIDC specific data and one with the ClientInfo. 
I'm not sure Hydra should handle ClientInfo part at all. It looks IdP specific and I know I will have to add some fields in my service and not use others. I might as well store my own version of ClientInfo